MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,200 @@
+
+N> Inherit the business object class class from the `INotifyPropertyChanged` for dynamic changes in custom data.
+
+## Handle appointment drag starting
+You can get the appointment details and handle whether the appointment can be draggable or not by using `AppointmentDragStarting` event. This event will be triggered when the appointment is started dragging. The `AppointmentDragStartingEventArgs` argument contains the following properties.
+
+`Appointment` - Gets the appointment that is going to be dragged.
+`Resource` - Gets the resource under which the appointment is currently located.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragStarting += OnSchedulerAppointmentDragStarting;
+
+...
+
+private void OnSchedulerAppointmentDragStarting(object? sender, AppointmentDragStartingEventArgs e)
+{
+    var appointment = e.Appointment;
+    e.Cancel = false;
+}
+{% endhighlight %}
+{% endtabs %}
+
+Refer the below code to cancel the all day appointment dragging.
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragStarting += OnSchedulerAppointmentDragStarting;
+
+...
+
+private void OnSchedulerAppointmentDragStarting(object? sender, AppointmentDragStartingEventArgs e)
+{
+    var appointment = e.Appointment as SchedulerAppointment;
+    if (appointment.IsAllDay)
+    {
+        e.Cancel = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Here, appointment dragging is disabled when the appointment is `AllDay` appointment.
+
+## Handle appointment dragging
+you can handle the dragging appointment details, position and time of the particular location by using `AppointmentDragOver` event. This event will be continuously triggered when the appointment is being dragged. The `AppointmentDragOverEventArgs` argument contains the following properties.
+
+`Appointment` - Gets the appointment that is being dragged.
+`DragPoint`- Gets the dragging point of the scheduler appointment user interface.
+`DragTime` - Gets the dragging time of the appointment object being dragged.
+`DragResource` - Gets the original resource of the appointment that is being dragged.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDragOver += OnSchedulerAppointmentDragOver;
+
+...
+
+private void OnSchedulerAppointmentDragOver(object? sender, AppointmentDragOverEventArgs e)
+{
+    var appointment = e.Appointment;
+    var draggingPoint = e.DragPoint;
+    var draggingTime = e.DragTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Handle appointment drop
+Using `AppointmentDrop` event you can get the dropping appointment details, position, time and you can handle whether the appointment can be dropped to the specific position or not. This event will trigger after dropping the appointment. The `AppointmentDropEventArgs` argument contains the following properties
+
+`Appointment` - Gets the appointment that is being dragged and dropped.
+`DropTime`- Gets or sets the date and time at which the appointment is being dropped.
```

Comment:
`DropTime`- Get or set the date and time at which the appointment is being dropped.

Reply:
Content changed